### ISSUE TITLE
For windows

### ISSUE THREAD

**Issue by ICH525** [nanoscopic/iosif#7]
It's better if you guys develop for windows also.

**Comment by nanoscopic** [comment]
Agreed; as discussed previously elsewhere ( I forget where exactly though ), this can be done by utilizing the mobiledevice framework that is within iTunes.

Besides having to build against that different library, the code should work otherwise the same.
